doc/classes/Callable.xml

```diff
@@ -160,7 +188,11 @@
 		<method name="is_custom" qualifiers="const">
 			<return type="bool" />
 			<description>
-				Returns [code]true[/code] if this [Callable] is a custom callable. Custom callables are created from [method bind] or [method unbind]. In GDScript, lambda functions are also custom callables.
+				Returns [code]true[/code] if this [Callable] is a custom callable. Custom callables are used:
```

Comment:
I may be better to reword the list to describe how custom Callables are created, how to create and obtain them, rather than list what they're used for.

I have no suggestions at the moment.